src/components/Title/index.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import styles from "./index.module.css";
+
+const Title = ({ children, theme, variants }) => {
+  return <div className={cx(styles[theme], styles[variants])}>{children}</div>;
```

Comment:
Remove the plural `variants` and use `variant`.

Them you use it as component to render the actual element:
```js
const Element = variant;

<Element {...etc tec}>{children}</Element>
```

Reply:
I think I don't get this one, what do I win in doing so? And how this will work for the themes?

Reply:
We need to render HTML text elements on these components and not `divs`.  With the new component-based approach you can just render an  `h2` here.

Why not a `h1`? Well, a single HTML page should only have **one** `h1` tag. So I would say this component should render a `h2` always, and we should use a custom prop to render a `h1` when we feel like it.

I usually use the `as` prop for that. It's an optional prop used to change the semantic element of a component.

So a `<Title>Meme</Title>` becomes `<h1>Title</h1>`

And `<Title as="h1">Meme</Title>` becomes `<h1>Meme</h1>`

The `as` prop just changes the element. Nothing more. It does not specify size, theme or anything like that. Just the element that appears on the DOM, which is super important for SEO and Accessibility.

Reply:
got it. Thanks
But, does it makes sense if we go with the approach of spliting this way: 
```
<Title>Minho Covid</Title>
<Subtitle>Ajudamos a ajudar</Subtitle>
<Text>Budaerros</Text>
```

Reply:
Yeah, if it's `divs` underneath then it's all the same